mypy/typeshed/stdlib/re.pyi

```diff
@@ -133,25 +133,19 @@ class Pattern(Generic[AnyStr]):
     @overload
     def fullmatch(self, string: AnyStr, pos: int = 0, endpos: int = sys.maxsize) -> Match[AnyStr] | None: ...
     @overload
-    def split(self: Pattern[str], string: str, maxsplit: int = 0) -> list[str | Any]: ...
```

Comment:
put comments that say these functions are intrinsically handled by mypy

Reply:
check the diff

Reply:
i think it should maybe have a comment on each one with special casing

Reply:
Hows that look?